client/containers/workflow/component.vue

```diff
@@ -144,17 +148,14 @@ export default {
       this.summary.result = undefined;
       this.summary.wfStatus = undefined;
       this.summary.workflow = undefined;
+
+      this.$emit('clearWorkflow');
```

Comment:
does it clean the wf state?

Reply:
correct. It will clear all data for the workflow in the vuex store.